src/pages/home/report/ReportActionsView.js

```diff
@@ -263,6 +237,10 @@ class ReportActionsView extends React.Component {
      * action when scrolled
      */
     recordMaxAction() {
+        if (this.unreadTimer) {
+            clearTimeout(this.unreadTimer);
+            this.unreadTimer = null;
+        }
```

Comment:
Would it be better to just have one method responsible for setting the timer instead of `recordMaxAction` having to clear these timers that are set in other places? e.g.  something like:

```js
function scheduleRecordMaxAction(delay = 0) {
    // Always cancel the existing timer
    clearTimeout(this.unreadTimer);
    this.unreadTimer = null;

    if (!delay) {
        this.recordMaxAction();
        return;
    }

    this.unreadTimer = setTimeout(this.recordMaxAction, delay);
}
```

Reply:
I like that!